crates/e2e/tests/e2e/partially_fillable.rs

```diff
@@ -0,0 +1,125 @@
+
+    let [solver] = onchain.make_solvers(to_wei(1)).await;
+    let [trader_a] = onchain.make_accounts(to_wei(1)).await;
+    let [token_a, token_b] = onchain
+        .deploy_tokens_with_weth_uni_v2_pools(to_wei(1_000), to_wei(1_000))
+        .await;
+
+    token_a.mint(trader_a.address(), to_wei(101)).await;
+    token_a.mint(solver.address(), to_wei(1000)).await;
+    token_b.mint(solver.address(), to_wei(1000)).await;
+
+    tx!(
+        solver.account(),
+        onchain
+            .contracts()
+            .uniswap_v2_factory
+            .create_pair(token_a.address(), token_b.address())
+    );
+    tx!(
+        solver.account(),
+        token_a.approve(
+            onchain.contracts().uniswap_v2_router.address(),
+            to_wei(1000)
+        )
+    );
+    tx!(
+        solver.account(),
+        token_b.approve(
+            onchain.contracts().uniswap_v2_router.address(),
+            to_wei(1000)
+        )
+    );
+    tx!(
+        solver.account(),
+        onchain.contracts().uniswap_v2_router.add_liquidity(
+            token_a.address(),
+            token_b.address(),
+            to_wei(1000),
+            to_wei(1000),
+            0_u64.into(),
+            0_u64.into(),
+            solver.address(),
+            U256::max_value(),
+        )
+    );
+
+    tx!(
+        trader_a.account(),
+        token_a.approve(onchain.contracts().allowance, to_wei(101))
+    );
+
+    let services = Services::new(onchain.contracts()).await;
+    services.start_autopilot(vec![
+        "--process-partially-fillable-limit-orders=true".to_string()
+    ]);
+    services
+        .start_api(vec![
+            "--allow-placing-partially-fillable-limit-orders=true".to_string()
+        ])
+        .await;
+
+    let order_a = OrderBuilder::default()
+        .with_sell_token(token_a.address())
+        .with_sell_amount(to_wei(100))
+        .with_buy_token(token_b.address())
+        .with_buy_amount(to_wei(80))
+        .with_valid_to(model::time::now_in_epoch_seconds() + 300)
+        .with_kind(OrderKind::Sell)
+        .with_partially_fillable(true)
+        .sign_with(
+            EcdsaSigningScheme::Eip712,
+            &onchain.contracts().domain_separator,
+            SecretKeyRef::from(&SecretKey::from_slice(trader_a.private_key()).unwrap()),
+        )
+        .build()
+        .into_order_creation();
+    services.create_order(&order_a).await.unwrap();
+
+    tracing::info!("Waiting for order to show up in auction.");
+    let has_order = || async { services.get_auction().await.auction.orders.len() == 1 };
+    wait_for_condition(TIMEOUT, has_order).await.unwrap();
+
+    let auction = services.get_auction().await.auction;
+    let order = auction.orders.into_iter().next().unwrap();
+    assert!(order.data.partially_fillable);
+    assert_eq!(order.metadata.full_fee_amount, 0.into());
+    assert_eq!(order.metadata.solver_fee, 0.into());
+    assert_eq!(auction.rewards.get(&order.metadata.uid), None);
```

Comment:
Note these asserts. Speak up if any of these values should be different.

Reply:
These seem correct to me:
- `partiallyFillable` is set
- There is no "full fee amount" or "solver fee" as this is dependent on how much the order executes and cannot be known upfront (cc @sunce86 - as this may affect CIP-20 computations as we rely on these fields - posted in Slack as well for more visibility).
- There are no CIP-14 risk-adjusted rewards for these orders (we explicitly decided not to figure out what we need to do for this as we plan on being on CIP-20 before partially fillable limit orders are available)

Reply:
I guess we could also assert that this was actually created as a limit order as opposed to a liquidity order.